gitbook-docs/security.md

```diff
@@ -112,6 +126,92 @@ Clients using other kinds of protocols can include the `token` in the Signal K m
 }
 ```
 
+### Token Validation
+
+Tokens may have a short expiry time and need to be renewed periodically, or a token's current validity may be unknown. 
```

Comment:
`expiry` is now available, so in what case is the validity unknown?

Reply:
Its possible for a token with valid expiry to be invalid for server-side reasons. eg The artemis server invalidates all tokens after a reboot, because the signing key is regenerated each boot.

Reply:
Leave the unknown validity out, as it does not have any bearing on either server or client implementation as far as I can tell?